deps/Makefile

```diff
@@ -30,7 +30,7 @@ CONFIGURE_COMMON += --host=$(XC_HOST)
 endif
 ifeq ($(OS),WINNT)
 ifneq ($(USEMSVC), 1)
-CONFIGURE_COMMON += LDFLAGS=-Wl,--stack,8388608
+CONFIGURE_COMMON += LDFLAGS="$(LDFLAGS) -Wl,--stack,8388608"
```

Comment:
I think your concern here is pointed more at `-Wl,--stack,8388608` than `$(LDFLAGS)`, since the change I've made here could only help that situation, right?